startleft/config/default-ctf-mapping.yaml

```diff
@@ -181,14 +189,199 @@ components:
+      - { $path: "Type" }
+
+  - id:     {$format: "{name}"}
+    type:   cf-cloudfront
+    name:   {$path: "_key"}
+    $source: {$singleton: {$root: "Resources|squash(@)[?starts_with(Type, 'AWS::CloudFront')]"}}
+    parent:  public-cloud
+    tags:
+      - { $path: "Type" }
+
+  - id:     {$format: "{name}"}
+    type:   cloudtrail
+    name:   {$path: "_key"}
+    $source: {$root: "Resources|squash(@)[?starts_with(Type, 'AWS::CloudTrail')]"}
+    parent:  public-cloud
+    tags:
+      - { $path: "Type" }
+
+  - id:     {$format: "{name}"}
+    type:   cognito
+    name:   {$path: "_key"}
+    $source: {$root: "Resources|squash(@)[?starts_with(Type, 'AWS::Cognito::UserPool')]"}
+    parent:  public-cloud
+    tags:
+      - { $path: "Type" }
+
+  - id:     {$format: "{name}"}
+    type:   cognito
+    name:   {$path: "_key"}
+    $source: {$root: "Resources|squash(@)[?starts_with(Type, 'AWS::Cognito::IdentityPool')]"}
+    parent:  public-cloud
+    tags:
+      - { $path: "Type" }
+
+  - id:     {$format: "{name}"}
+    type:   CD-CONFIG
+    name:   {$path: "_key"}
+    $source: {$singleton: {$root: "Resources|squash(@)[?starts_with(Type, 'AWS::Config')]"}}
+    parent:  public-cloud
+    tags:
+      - { $path: "Type" }
+
+  - id:     {$format: "{name}"}
+    type:   elastic-container-registry
+    name:   {$path: "_key"}
+    $source: {$singleton: {$root: "Resources|squash(@)[?starts_with(Type, 'AWS::ECR')]"}}
+    parent:  public-cloud
+    tags:
+      - { $path: "Type" }
+
+  - id:     {$format: "{name}"}
+    type:   elastic-container-kubernetes
+    name:   {$path: "_key"}
+    $source: {$root: "Resources|squash(@)[?starts_with(Type, 'AWS::EKS::Cluster')]"}
+    parent:  public-cloud
+    tags:
+      - { $path: "Type" }
+
+  - id:     {$format: "{name}"}
+    type:   elasticache
+    name:   {$path: "_key"}
+    $source: {$singleton: {$root: "Resources|squash(@)[?starts_with(Type, 'AWS::ElastiCache')]"}}
+    parent:  public-cloud
+    tags:
+      - { $path: "Type" }
+
+  - id:     {$format: "{name}"}
+    type:   CD-GUARDDUTY
+    name:   {$path: "_key"}
+    $source: {$singleton: {$root: "Resources|squash(@)[?starts_with(Type, 'AWS::GuardDuty')]"}}
+    parent:  public-cloud
+    tags:
+      - { $path: "Type" }
+
+  - id:     {$format: "{name}"}
+    type:   CD-INSPECTOR
+    name:   {$path: "_key"}
+    $source: {$singleton: {$root: "Resources|squash(@)[?starts_with(Type, 'AWS::Inspector')]"}}
+    parent:  public-cloud
+    tags:
+      - { $path: "Type" }
+
+  - id:     {$format: "{name}"}
+    type:   CD-MACIE
+    name:   {$path: "_key"}
+    $source: {$singleton: {$root: "Resources|squash(@)[?starts_with(Type, 'AWS::Macie')]"}}
+    parent:  public-cloud
+    tags:
+      - { $path: "Type" }
+
+  - id:     {$format: "{name}"}
+    type:   CD-AWS-NETWORK-FIREWALL
+    name:   {$path: "_key"}
+    $source: {$root: "Resources|squash(@)[?starts_with(Type, 'AWS::NetworkFirewall::Firewall')]"}
+    parent:  public-cloud
```

Comment:
I think that the parent should be taken from the vpcid, or if not possible maybe we can declare this as a singleton because it doesn't makes sense to have multiple firewalls in the threatmodel that are not acting against any object. What do you think?